sdk/core/core-xml/src/xml.common.ts

```diff
@@ -26,4 +26,14 @@ export interface XmlOptions {
    * key used to access the XML value content when parsing XML.
    */
   xmlCharKey?: string;
+
+  /**
+   * property name for a CDATA section.
+   */
+  cdataPropName?: string;
+
+  /**
+   * XML nodes to exclude from parsing.
+   */
+  stopNodes?: string[];
```

Comment:
I wonder if this name is clear enough or another could be clearer? excludedNodes?

Reply:
The name isn't great but it looks like this comes from our underlying dependency: https://github.com/NaturalIntelligence/fast-xml-parser/blob/master/docs/v4/3.XMLBuilder.md#stopnodes

I think it's fine enough since it's commented